openpype/modules/deadline/plugins/publish/submit_nuke_deadline.py

```diff
@@ -123,13 +121,10 @@ def process(self, instance):
         render_path = instance.data['path']
         script_path = context.data["currentFile"]
 
-        for item in context:
-            if "workfile" in item.data["families"]:
-                msg = "Workfile (scene) must be published along"
-                assert item.data["publish"] is True, msg
```

Comment:
This not being forced anymore seems like quite a change. Who might have been relying on this?